### PR TITLE
Fix add mac os version

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

Add macOS version to wizard deploy agent
 
### Issues Resolved
[#4204][#4178]

### Evidence
image

### Steps to test:
*Go to the Agents tab
*Click on the 'Deploy new agent' button.
*Verify that the macOS Ventura version is displayed.

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Tostti — APPROVED** [review]
LGTM

**Review by asteriscos — CHANGES_REQUESTED** [review]
When I select any macOS it still asks for an architecture. 
Peek 2022-11-15 12-35

**Review by Tostti — APPROVED** [review]
Tested latest changes

File: public/controllers/agent/components/register-agent.js [hunk 1]
```diff
@@ -195,7 +195,7 @@ export const RegisterAgent = withErrorBoundary(
     }
 
     systemSelectorWazuhControlMacos() {
-      if (this.state.selectedVersion == 'sierra' || this.state.selectedVersion == 'highSierra' || this.state.selectedVersion == 'mojave' || this.state.selectedVersion == 'catalina' || this.state.selectedVersion == 'bigSur' || this.state.selectedVersion == 'monterrey') {
+      if (this.state.selectedVersion == 'sierra' || this.state.selectedVersion == 'highSierra' || this.state.selectedVersion == 'mojave' || this.state.selectedVersion == 'catalina' || this.state.selectedVersion == 'bigSur' || this.state.selectedVersion == 'monterrey' || this.state.selectedVersion == 'ventura') {
```
Comment (asteriscos) [inline public/controllers/agent/components/register-agent.js:198]: maybe this conditional could be simplified to improve readability and maintainability. Perhaps something like:
`macOSversionList.includes(this.state.selectedVersion)`

**Review by asteriscos — APPROVED** [review]
CR: :heavy_check_mark: 
Test: :heavy_check_mark: 
screencapture-localhost-5601-app-wazuh-2022-11-15-12_53_32

File: public/controllers/agent/components/register-agent.js (hunk 1 above)
Comment (chantal-kelm) [reply public/controllers/agent/components/register-agent.js:198]: Yes, I understand that this is already being done in the logic improvement issue.

**Pull request merged**